external_distributions/mumps_and_scalapack/Makefile.am

```diff
@@ -127,13 +127,16 @@ endif
 	echo "#Install with blacs (downloaded on the fly)" >> build_script.bash
 	echo "echo \"Bypassing test of executables (because it breaks on hector, or any\"">> build_script.bash
 	echo "echo \"other machine where we cross compile)\"">> build_script.bash
-	echo "cp script/framework.py script/framework.py.orig" >> build_script.bash
-	echo "sed 's/.\/tmpf/date/g' script/framework.py.orig | sed 's/.\/tmpc/date/g' > script/framework.py ">> build_script.bash
-	echo "patch script/blacs.py < ../../build_scripts/blacs.py.patch  ">> build_script.bash
+#OLD (PYTHON2)	echo "cp script/framework.py script/framework.py.orig" >> build_script.bash
+	echo "cp ../../scalapack_python3_scripts/*py script" >> build_script.bash
+#OLD (PYTHON2)	echo "sed 's/.\/tmpf/date/g' script/framework.py.orig | sed 's/.\/tmpc/date/g' > script/framework.py ">> build_script.bash
+#OLD (PYTHON2)	echo "patch script/blacs.py < ../../build_scripts/blacs.py.patch  ">> build_script.bash
```

Comment:
Might as well kill the lines with `#OLD (PYTHON2)`, we can always flick back through them with Git.

Reply:
If you don't mind, I'll keep this in. It's true that flicking back through the history is possible but the way this is being customised/built is not straightforward and this comment will remind me (without having to consult extra information) about the logic. In the current state I'm likely to be the only one to go one there if stuff breaks again, and hopefully it'll all be superseded by lightning fast cmake installs anyway and they're (hopefully) bypassing all this horror.